CDCgov/NEDSS-Modernization
# CNFT1-1226 Patient Profile Multiple Races

## Description

Displays more than one race on Patient Profile Summary Card in accordance with the rules to display race values in NBS Classic.  A value of `Unknown` is shown only if a "NullFlavor" race is not present.

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/124325935/68e22499-c430-449d-b791-55872632e991)


## Tickets

* [CNFT1-1226](https://cdc-nbs.atlassian.net/browse/CNFT1-1226)

## Steps to verify

1. Navigate to a Patient Profile without a Race
2. Navigate to the Demographics tab
3. Add a Race of `Unknown`
4. The Race should appear in the top of the Patient Profile 
5. Add a Race of 'Not asked`
6. The Race should remain as `Unknown` in the top of the Patient Profile 
7. Add a Race of one of `American Indian or Alaska Native`, `Asian`, `Black or African American`, `Native Hawaiian or Other Pacific Islander`, `White`, or `Other`
8. The Race should be displayed in the top of the Patient Profile 
9. The Race should no longer have a value of `Unknown`


[CNFT1-1226]: https://cdc-nbs.atlassian.net/browse/CNFT1-1226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
